src/contacts/ContactAside.tsx

```diff
@@ -40,17 +43,31 @@ export const ContactAside = ({ link = 'edit' }: { link?: 'edit' | 'show' }) => {
             </Box>
             <Typography variant="subtitle2">Personal info</Typography>
             <Divider sx={{ mb: 2 }} />
-            {record.email && (
-                <Stack
-                    direction="row"
-                    alignItems="center"
-                    gap={1}
-                    minHeight={24}
-                >
-                    <EmailIcon color="disabled" fontSize="small" />
-                    <EmailField source="email" />
-                </Stack>
-            )}
+            <ArrayField source="email_jsonb">
+                <SingleFieldList linkType={false} gap={0} direction="column">
+                    <Stack
+                        direction="row"
+                        alignItems="center"
+                        gap={1}
+                        minHeight={24}
+                    >
+                        <EmailIcon color="disabled" fontSize="small" />
+                        <Box>
+                            <EmailField source="email" />{' '}
+                            <WithRecord
+                                render={row =>
+                                    row.type !== 'Other' && (
+                                        <TextField
+                                            source="type"
+                                            color="textSecondary"
+                                        />
+                                    )
+                                }
+                            />
+                        </Box>
```

Comment:
Use flexbox instead of a space? This ensure better compatibility with custom themes.